src/api/pki/responses.rs

```diff
@@ -6,6 +6,7 @@ use serde::{Deserialize, Serialize};
 pub struct GenerateCertificateResponse {
     pub ca_chain: Option<Vec<String>>,
     pub certificate: String,
+    pub expiration: Option<i64>,
```

Comment:
The `expiration` should never be a negative number.
```suggestion
    pub expiration: Option<u64>,
```